tsconfig.json

```diff
@@ -5,7 +5,7 @@
     "declarationDir": "./dist/lib",
     "noImplicitAny": false,
     "target": "es2015",
-    "sourceMap": false,
+    "sourceMap": true,
```

Comment:
Why do we need to enable sourceMaps?

Reply:
Yes, the `.map` files get generated for each `.js` file on `dist` folder. Why do you think sourceMaps should not be generated?

Reply:
If we want to enable source maps, we cannot do so for productive builds as this exposes the code to be visible to all the users of content-cli which typically exposes us to the risk of reverse engineering. These rules do not apply as strictly for open source projects, however if we only need it for debugging, we should use sourceMaps only during the development builds and not productive ones.

Reply:
@qmucolli I set `sourceMap` to true only when running Jest tests in [this commit](https://github.com/celonis/content-cli/pull/184/commits/139cb2bf090f8c3ef0f9a30cd2013fcb803af3bd). This works when debugging the code and doesn't generate source maps when running `yarn build`. Can you check it out?